src/appModel.ts

```diff
@@ -70,8 +78,6 @@ export class AppModel {
      */
     async compileAllFiles() {
         try {
-            StatusBarUi.working();
-
```

Comment:
Re-add this. [ref](https://github.com/glenn2223/vscode-live-sass-compiler/pull/3#issuecomment-748591053)

Reply:
Done